src/components/Books/Book.js

```diff
@@ -0,0 +1,32 @@
+import { CircularProgressbar } from 'react-circular-progressbar';
+
+const Book = () => {
```

Comment:
- [ ] The `Book` component takes the book data as a prop in order to use them to display book data in the template. Let's fix that